docs-source/spring/content/development/cli/_index.md

```diff
@@ -25,19 +23,80 @@ Table of Contents:
   * [List - show details about deployments](#list)
   * [Manage config server data](#config)
   * [GraalVM compile commands](#compile)
+  * [User Management](#user-management)  
+* [Logging Information](#logging)
+
+## Installation
+
+1. CLI is available for Linux on x64 and AArch64, macOS on x64 and AArch64, and Windows x64 architectures. Choose your operating system and proceed to the installation steps for your specific platform:
```

Comment:
The second sentence is confusing - I don't see any links to os-specific installation steps?  Are there even any, don't you just download it and run it?